### PR TITLE
WIP: Document ability to import & export content libraries

### PR DESCRIPTION
**Pull request by catong**
This PR documents the ability to import and export content libraries SOL-82/SOL-368.
@antoviaque @Kelketek @marcotuts can you please review for technical accuracy? Thanks!

### CODE REVIEW

**Comment by marcotuts** [comment]
(Looks like this might need a rebase, but otherwise) :+1:

**Comment by catong** [comment]
@mhoeber @lamagnifica @srpearce Could one of you please do an editorial review on this PR?

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 1]
```diff
@@ -387,3 +389,92 @@ To remove **Staff** or **Admin** access from a library user, follow these steps.
...
+You might want to import a library if you developed or updated library content
+outside of Studio, or if you want to overwrite a problematic or outdated version
+of the library.
+
+.. warning:: When you import a library, it completely replaces the existing
+   library and its contents. You cannot undo a library import. Before you
+   proceed, we recommend that you export the current library, so that you
+   have a backup copy of it.
+
+The library file that you import must be a .tar.gz file (that is, a .tar file compressed using GNU Zip). This .tar.gz file must contain a library.xml file.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst]: Line wrapping

Also, is there a reason you explain what a .tar.gz file is here but not in the export section?

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 2]
```diff
@@ -387,3 +389,92 @@ To remove **Staff** or **Admin** access from a library user, follow these steps.
...
+================
+Import a Library
+================
+
+You might want to import a library if you developed or updated library content
+outside of Studio, or if you want to overwrite a problematic or outdated version
+of the library.
+
+.. warning:: When you import a library, it completely replaces the existing
+   library and its contents. You cannot undo a library import. Before you
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst]: I'd change "it" to "the imported library"

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 3]
```diff
@@ -387,3 +389,92 @@ To remove **Staff** or **Admin** access from a library user, follow these steps.
...
+To import a library, follow these steps.
+
+#. In Studio, select the **Libraries** tab. 
+   
+#. Locate the library to which you want to import the new library content.
+    
+#. From the **Tools** menu, select **Import**.
+   
+#. Click **Choose a File to Import** and select the .tar.gz file that you are
+   importing.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst]: Style thing - I'd avoid progressive tense. So I'd change this to "select the .tar.gz file that you want to import."

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 4]
```diff
@@ -387,3 +389,92 @@ To remove **Staff** or **Admin** access from a library user, follow these steps.
...
+
+#. Click **Replace my library with the selected file**.
+   
+.. warning:: The import process has five stages. During the first two stages
+   (Uploading and Unpacking), do not navigate away from the Library Import page.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst]: **Library Import** page

File: en_us/shared/building_and_running_chapters/creating_content/libraries.rst [hunk 5]
```diff
@@ -387,3 +389,92 @@ To remove **Staff** or **Admin** access from a library user, follow these steps.
...
+   Doing so causes the import process to end. You can leave the page only after
+   the Unpacking stage completes. We recommend that you do not make important
+   changes to the library until all stages of the import process have finished.
+
+5. When the import process finishes, click **View Updated Library** to view the
+   imported library.
+
+.. note:: If your imported library includes changes to components that are in
+   use in a course, these updates in the library are not reflected in the course
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/creating_content/libraries.rst]: "the course does not reflect these library updates until"

**Comment by srpearce** [comment]
A few nits - otherwise :+1:

**Comment by Kelketek** [comment]
@catong This looks good to me! :+1:

**Comment by catong** [comment]
@marcotuts @Kelketek I thought I would keep it simple re documenting Show/Hide Preview - please take a look at https://github.com/edx/edx-documentation/commit/5acfd4ee5bd8c11011273f8b18b940278c098418 and let me know if this is adequate? Thanks!
NB -- if we are adding ability to expand single components at a time, I'll make modifications.

**Comment by catong** [comment]
Accidentally closed. These commits moved to new PR https://github.com/edx/edx-documentation/pull/204

**Pull request closed without merging**